### DISCUSSION TITLE
Displaying a list of all windows when calling the Flow-Launcher by default

### DISCUSSION THREAD

**Discussion by vbccbz** [Flow-Launcher/Flow.Launcher#2579] [Ideas]
Windows Walker is brilliant, but it would be nice to have list of all opened window simply by calling Flow-Launcher without typing.

**Comment by vbccbz** [comment]
Windows Walker / action / tab
Last Query Style / Select last query

**Comment by VictoriousRaptor** [comment]
or you can combine custom query hotkey with window walker's keyword

**Reply by vbccbz** [reply]
Windows Walker / action / tab
Custom Query Hotkeys / Hotkey Alt + Return , query tab

**Comment by arthur-er** [comment]
it's possible to do it without any keyword so it shows all tabs by default but allows me to start typing right away?
